mobile/features/create/CreateWorkout.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+import { NativeStackNavigationProp } from '@react-navigation/native-stack';
+import { RootStackParamList } from '../../navigation/NavigationConstants';
+import { RouteProp } from '@react-navigation/native';
+
+type CreateWorkoutNavigationProp = NativeStackNavigationProp<
+  RootStackParamList,
+  'CreateWorkout'
```

Comment:
I'm really not satisfied with `CreateWorkout` and other nav screen names being defined as magic strings and repeated. but that's how they do it here: https://reactnavigation.org/docs/typescript/

will need to revisit this before the project scales too much.